pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -130,6 +131,9 @@ func Run() {
 	}
 	pubUuidToNum.ClearRestarted()
 
+	// Create the dummy interface used to re-direct DROP/REJECT packets.
+	createFlowMonDummyInterface(0xFFFFFFFF)
```

Comment:
Should 0xFF... be a defined constant somewhere?

Reply:
Yes, will do.